client/components/tinymce/plugins/insert-menu/plugin.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import tinymce from 'tinymce/tinymce';
+import { renderToString } from 'react-dom/server';
+
+import Gridicon from 'components/gridicon';
+import i18n from 'lib/mixins/i18n';
+
+import menuItems from './menu-items';
+
+const initialize = editor => {
+	menuItems.forEach( item =>
```

Comment:
this was part of the call on how many places to scatter the config flag check. I felt that adding was light-weight enough to leave it in. if you want, I'll hide it behind the flag.